datacollection/models.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+
+# Create your models here.
+from django.db.models import Count
+from django.urls import reverse
+
+from database.models.structural_models import ModelSet
+
+
+class PreferenceForm(models.Model):
+    """
+    Keeps Track of preference form Open for a set.
+    """
+    set = models.ForeignKey(ModelSet, on_delete=models.CASCADE, related_name='preference_form')
+    name = models.CharField('Form Name', max_length=100)
+    is_taking_responses = models.BooleanField(default=True)
+    is_active = models.BooleanField(default=True)
+
+    def __str__(self):
+        return f'{self.set} --> {self.is_taking_responses}'
+
+    @property
+    def total_students(self):
+        from database.models.user_models import Student
```

Comment:
this seems to be a weird place to import something - shouldn't this be at the top of the file?